### ISSUE TITLE
Export Results to comma separated text

### ISSUE THREAD

**Issue by joebenimble** [splitbrain/dokuwiki-plugin-data#107]
The ability to export the results would be great.

**Comment by FlominatorTM** [comment]
Indeed. Until then: Select the table and copying it to LibreOffice Calc works ok as well.

**Comment by piftouf** [comment]
I definitely agree, export possibilities would be great for this plugin!
